gaurun/log.go

```diff
@@ -45,7 +46,11 @@ type Reopener interface {
 	Reopen() error
 }
 
-func InitLog(outString, levelString string) (zap.Logger, Reopener, error) {
+func LocalTimeEncoder(t time.Time, enc zapcore.PrimitiveArrayEncoder) {
+	enc.AppendString(t.Format("2006/01/02 15:04:05 MST"))
+}
+
+func InitLog(outString, levelString string) (*zap.Logger, Reopener, error) {
```

Comment:
[nit] It's ok for now but for next time, 

- I will avoid to pass `outString` but `interface`
- If I can not avoid it to pass `outString`, at least, I prepare `const` for "stdout"/"stderr"/etc to avoid mismatch